build.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs-extra");
+const generateHTMLs = require("./utils/generateHTMLs.js");
+const config = require("./config.js");
+
+function clearBuildPath(buildPath){
+  try {
```

Comment:
Everything under the Joplin org needs to use tabs, not spaces.

Reply:
Done